easybuild/easyconfigs/h/hwloc/hwloc-1.11.10-GCCcore-7.3.0.eb

```diff
@@ -25,15 +25,20 @@ source_urls = ['https://www.open-mpi.org/software/hwloc/v%(version_major_minor)s
 sources = [SOURCE_TAR_GZ]
 checksums = ['0a2530b739d9ebf60c4c1e86adb5451a20d9e78f7798cf78d0147cc6df328aac']
 
+osdependency = [('libudev-dev', 'libudev-devel')]
```

Comment:
@akesandgren As discussed on Slack, I'm not sure this is a good idea...

As I understand, `libudev` is an optional dependency for `hwloc`? Including it as OS dep makes it mandatory (unless `--ignore-osdeps` is used).

Also, it seems like on `CentOS` this should be `libgudev1-devel`?

Reply:
libgudev1-devel is udev for those that use glib, libudev-devel is the normal one.

Reply:
And you said: "hmm, if it's an optional dep, we should include it as an OS dep since that'll make it a hard requirement" in the channel... so i included it...

Reply:
We don't have `libudev-devel` installed on our systems, so in our case adding this OS dep would be problematic.

W.r.t. the statement on Slack: I clearly hadn't have enough coffee (or wine...) yet at that point. What I meant to say was that we shouldn't make `libudev-devel` as hard requirement by listed it as an OS dep if it's actually an optional dependency...